downloader.py

```diff
@@ -90,28 +96,28 @@ def download_file(filename, prefix, num_videos=None):
 
             itag = filtered.first().itag
             filename = '{}.mp4'.format(video_id)
-            filename_mp4 = '{}.mp4'.format(video_id)
-            original_video_location = f'videos/{filename}'
-            keyframe_video_location = f'videos/key-{filename}'
+            original_video_location = f'{prefix}_videos/{filename}'
+            keyframe_video_location = f'{prefix}_videos/key-{filename}'
             download_video(url, itag, filename=original_video_location)
 
             start_min = start_time//60
             start_sec = start_time%60
-            subprocess.call(f'ffmpeg -i {original_video_location} -force_key_frames 00:{start_min}:{start_sec} {keyframe_video_location}', shell=True)
-            ffmpeg_extract_subclip(keyframe_video_location,
-                    start_time,
-                    end_time,
-                    targetname='{}_videos/{}'.format(prefix, filename_mp4))
+            subprocess.call(f'ffmpeg -y -i {original_video_location} -force_key_frames 00:{start_min}:{start_sec} {keyframe_video_location}', shell=True)
+            #ffmpeg_extract_subclip(keyframe_video_location,
+            #        start_time,
+            #        end_time,
+            #        targetname=filename_mp4)
+
+            subprocess.call(f'ffmpeg -y -ss {start_time} -i {keyframe_video_location} -t 1 -vcodec copy -acodec copy -y {filename_mp4}', shell=True)
 
             extract_audio(prefix, video_id)
 
-            os.remove(original_video_location)
+            #os.remove(original_video_location)
```

Comment:
Why has this line been commented out? I'm worried that this will result in a lot of storage used.

Reply:
I found that the original code was actually deleting all video clips, so the resulting directory was empty. I think this may be because filename was the same as filename_mp4.

Reply:
Hmm, was it deleting the shortened version as well? That's really strange if it was.
In any case, if it works, nbd.